### ISSUE TITLE
Pink Signal Lines and Magnetic Fields when using DirectX 11

### ISSUE THREAD

**Issue by DeltaDizzy** [ScienceFun/Kerbalism-Continued#12]
Whenever I use Kerbalism with DX11, planet magnetospheres and signal lines are bright pink. I know it isn't just my install because someone else has experienced it as well. If you need logs or any other files then I can provide them. This issue was orignally opened here: https://github.com/ShotgunNinja/Kerbalism/issues/166

**Comment by JotaRata** [comment]
Yeah, I've posted this already in the old Kerbals thread (https://github.com/ShotgunNinja/Kerbalism/issues/166) it's good to know that I'm not the only one to have this issue, hope it'll fixed soon

**Comment by HaullyGames** [comment]
Could you guys send me a screenshot?

**Comment by HaullyGames** [comment]
Hey guys, I googled a little bit and I found many articles saying that issue is about Unity with Directx11.

I found pictures with the same error (it is not only for Kerbalism).
[Pink texture issue](http://image.noelshack.com/fichiers/2016/17/1461672879-screenshot1.png)
[Forum](https://forum.kerbalspaceprogram.com/index.php?/topic/95837-forcing-directx11-40-memory-reduction/)

Looks like has some related with the parameter: **force-d3d11-no-singlethreaded**
Because this is not a Kerbalism issue, I'm closing this issue.

**Comment by JotaRata** [comment]
Yeah but I suggest in my post that you could add some integration with DX11 
As I learnt from Unity, maybe your shader may have Sub programs which are not compatible with DX11 and those would have to be fixed

**Comment by HaullyGames** [comment]
Ok, I will review the code.
Lines are created through code.

Could you confirm if you have any issue with RemoteTech's lines?

**Comment by JotaRata** [comment]
RemoteTech lines works perfectly with DX11. Sorry for the delay
